research/src/sources/primer.json5

```diff
@@ -97,6 +97,10 @@
           "description": "Use .hidden-text-expander to indicate and toggle hidden text."
         }
       ]
-    }
+    },
+    {
+      "name": "Flex",
```

Comment:
Is there a definition you could add here like the one `Button` has above?

Reply:
I can add something like:
`Flex provides a more efficient way to lay out, align and distribute space among items in a container, even when their size is unknown and/or dynamic` based on: https://css-tricks.com/snippets/css/a-guide-to-flexbox/ under the background section.